lib/commands.py

```diff
@@ -657,10 +657,23 @@ def is_synchronized(self):
         return self.wallet.is_up_to_date()
 
     @command('n')
-    def getfeerate(self):
-        """Return current optimal fee rate per kilobyte, according
-        to config settings (static/dynamic)"""
-        return self.config.fee_per_kb()
+    def getfeerate(self, fee_method=None, fee_level=None):
+        """Return current suggested fee rate (in sat/kvByte), according to config
+        settings or supplied parameters.
+        """
+        if fee_method is None:
+            dyn, mempool = None, None
+        elif fee_method == 0:  # static
+            dyn, mempool = False, False
+        elif fee_method == 1:  # ETA
+            dyn, mempool = True, False
+        elif fee_method == 2:  # mempool
+            dyn, mempool = True, True
```

Comment:
Why not dispense with the magic integers and just use the strings 'static'/'ETA'/'mempool'?

Reply:
good point